clusters/build-clusters/01_cluster/assets/github-secret_secret_template.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Template
+objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: github-secret
```

Comment:
nit: any reason not calling it `github-client-secret` so it hints what it specifically is?

Reply:
Changed.
No reason, just blindly copied from the doc. :)